### PR TITLE
Add TVC Share My Salary ULP

### PR DESCRIPTION
**Pull request by stephenmcmurtry**

### CODE REVIEW

File: frontend/content/press/releases/tvc-salary-sharing-ulp.en.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+“Every worker has the right to share their salaries and wages in order to ensure that we are all getting our fair share. When we had access to a shared spreadsheet we could see how across companies, workers were being paid vastly different amounts for their work. This is powerful information that we have a right to know and hope Alphabet does the right thing and returns access to all workers immediately. Alphabet and its many subcontractors must also commit to ending this practice of silencing workers attempting to discuss our wages and benefits on the job,” **said Laura Greene, worker with Accenture Flex and member of the Alphabet Workers Union - Communications Workers of America (AWU-CWA).**
```
Comment (realjenny) [inline frontend/content/press/releases/tvc-salary-sharing-ulp.en.md:21]: The person doesn't need to be bolded

File: frontend/content/press/releases/tvc-salary-sharing-ulp.en.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+On July 11, 2022 and July 14, 2022 Alphabet withdrew access to a spreadsheet. AWU-CWA estimates that as many as 50,000 workers are now locked out of the file. Workers employed directly by Alphabet continue to have access to an employee-run "Share My Salary" spreadsheet.
+
+“Once again, Alphabet is violating its own commitments to end the practice of preventing workers from discussing their wages and other working conditions. It’s clear that Alphabet and its various affiliates do not want workers to be armed with knowledge regarding pay rates across the company. Every Alphabet worker, including Temporary, Vendor and Contract workers, have a right to pay transparency and fair wages. We will continue to support workers fighting to exercise their basic rights on the job,” **said Shelby Hunter, AWU-CWA Organizing Chair and Google Maps worker with Cognizant.**
```
Comment (realjenny) [inline frontend/content/press/releases/tvc-salary-sharing-ulp.en.md:27]: The person doesn't need to be bolded

File: frontend/content/press/releases/tvc-salary-sharing-ulp.en.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+“Every worker has the right to share their salaries and wages in order to ensure that we are all getting our fair share. When we had access to a shared spreadsheet we could see how across companies, workers were being paid vastly different amounts for their work. This is powerful information that we have a right to know and hope Alphabet does the right thing and returns access to all workers immediately. Alphabet and its many subcontractors must also commit to ending this practice of silencing workers attempting to discuss our wages and benefits on the job,” **said Laura Greene, worker with Accenture Flex and member of the Alphabet Workers Union - Communications Workers of America (AWU-CWA).**
+
+In 2021, Temporary, Vendor and Contract (TVC) workers at Alphabet created a “Share My Salary” spreadsheet where contract workers across the globe could submit their job titles and hourly wages or salaries. Hundreds of workers submitted information in an effort to arm their fellow workers with necessary information regarding pay ranges for various positions.
```
Comment (realjenny) [inline frontend/content/press/releases/tvc-salary-sharing-ulp.en.md:23]: temporary, vendor, and contract

**Review by realjenny — CHANGES_REQUESTED** [review]

**Review by realjenny — APPROVED** [review]

File: frontend/content/press/releases/tvc-salary-sharing-ulp.en.md (hunk 1 above)
Comment (stephenmcmurtry) [reply frontend/content/press/releases/tvc-salary-sharing-ulp.en.md:21]: Word. That's how Catalina sends them out, so wasn't sure if we needed to preserve that.

**Pull request merged**